### DISCUSSION TITLE
A cpp program running on a Intel(R) Core(TM) i7-14700HX processor, how to set number of threads involved?

### DISCUSSION THREAD

**Discussion by xiaorong117** [scivision/mumps-superbuild#92] [Q&A]
as title says

**Comment by scivision** [comment]
Are you using OpenMP? I usually don't, but perhaps it would be the standard environment variable method. Otherwise, it would be a question for the MUMPS User Group email https://listes.ens-lyon.fr/sympa/subscribe/mumps-users
